src/test/java/wolox/training/controllers/BookControllerTest.java

```diff
@@ -179,6 +175,17 @@ void whenFindBookByIsbnThenRetunrStatusCreated() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON))
                 .andDo(print())
                 .andExpect(status().isCreated());
+    }
 
+    @WithMockUser(value = "miguel")
+    @Test
+    @DisplayName("Test , When a book is seached by publisher , genre and year ,it return status OK")
+    void whenFindByPublisherGenreAndYearThenReturnStatusOK() throws Exception {
```

Comment:
You should test it when book is not found

Reply:
corrected